assets/app/view/game/corporation.rb

```diff
@@ -45,53 +37,46 @@ def render
 
         children << render_companies if @corporation.companies.any?
 
-        revenue_table_props = {
-          style: {
-            'text-align': 'center',
-            'font-weight': 'bold',
-            'margin-left': 'auto',
-            'margin-right': 'auto',
-          },
-        }
-
         if @corporation.owner
           subchildren = (@corporation.operating_history.empty? ? [] : [render_revenue_history])
-          children << h(:table, revenue_table_props, [h(:tr, subchildren)])
+          children << h('table.revenue', [h(:tr, subchildren)])
         end
 
-        h(:div, { style: card_style, on: { click: onclick } }, children)
+        h('div.corp.card', { style: card_style, on: { click: onclick } }, children)
       end
 
       def render_title
-        title_style = {
-          background: @corporation.color,
-          color: @corporation.text_color,
-          'text-align': 'center',
-          padding: '0.5rem 0px',
-          'font-weight': 'bold',
-          margin: '-0.5rem -0.5rem 0 -0.5rem',
+        title_row_props = {
+          style: {
+            'background-color': @corporation.color,
+            color: @corporation.text_color,
```

Comment:
move back to inline style, i don't see much benefit to having a specific styling like token outside of here and in main.css